StytchDemo/B2BWorkbench/AuthMethodControllers/SessionsViewController.swift

```diff
@@ -37,14 +53,36 @@ final class SessionsViewController: UIViewController {
     private lazy var revokeAction: UIAction = .init { _ in
         Task {
             do {
-                let resp = try await StytchB2BClient.sessions.revoke()
-                print(resp)
+                let response = try await StytchB2BClient.sessions.revoke()
+                print("revokeAction response: \(response)")
             } catch {
                 print("revokeAction error: \(error.errorInfo)")
             }
         }
     }
 
+    private lazy var exchangeSessionsAction: UIAction = .init { _ in
+        self.exchangeSession()
+    }
+
+    func exchangeSession() {
+        guard let organizationID = orgIdTextField.text else {
+            return
+        }
+        Task {
+            do {
+                let parameters = Sessions<B2BAuthenticateResponse>.ExchangeParameters(organizationID: organizationID)
```

Comment:
Whats with the `<B2BAuthenticateResponse>`? Is it because it's using the Session client defined in StytchClientCommon? Should we go ahead and break out the session clients into separate ones for B2B and B2C? that way we could do `StytchB2BClient.Sessions.ExchangeParameters()`, right? Just for consistency with all the other products/methods. I think this current approach might lead to developer confusion.

Reply:
I personally feel that it is good the way it is and a good use of swift generics. If I can figure out something with sorcery where I can add a "where" clause to the protocol extensions then we can remove this. 

This is the way around this: 
`public extension Sessions where AuthResponseType == B2BAuthenticateResponse`

Then:
`Sessions<B2BAuthenticateResponse>.ExchangeParameters`
turns into:
`Sessions.ExchangeParameters` 

But currently I can't figure out how to do that if sorcery and I posted a question about how to do that for the generated files here: https://github.com/krzysztofzablocki/Sourcery/issues/1337.

I would only think we need to name space `Session` if it was to be used in tandem with the B2C session.

Reply:
If we split out the sessions clients (which we do on Android), would that avoid the unknown sourcery stuff? If so, that plus being able to namespace the parameters seems like a win to me. We already have some customer confusion with using two different clients, I just think that this might add to that cognitive load.

Reply:
Agreed, after thinking about it and pair programming, I think we do need separate objects for b2b session and b2c session, lets make a ticket to capture this work.